pkg/adapter/adapter.go

```diff
@@ -60,8 +60,14 @@ func NewLocalAdapter(cfg *config.Config) (*LocalAdapter, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "invalid platform configuration")
 	}
-
-	provider, content, err := content.NewLocalProvider(cfg.Provider.WorkDir, cfg.Provider.GCPolicy.Threshold, cfg.Host, platformMC)
+	useRemoteCache := false
+	for _, rule := range cfg.Converter.Rules {
+		if rule.CacheTagSuffix != "" {
+			useRemoteCache = true
+			break
+		}
+	}
+	provider, content, err := content.NewLocalProvider(cfg.Provider.WorkDir, cfg.Provider.GCPolicy.Threshold, cfg.Host, platformMC, useRemoteCache, cfg.Provider.CacheSize)
```

Comment:
Maybe we can simply the method to `content.NewLocalProvider(cfg)`, and provide a new method `cfg.EnableRemoteCache()`?

Reply:
fixed